demos/security_barrier_camera_demo/cpp/main.cpp

```diff
@@ -292,6 +292,10 @@ ReborningVideoFrame::~ReborningVideoFrame() {
         context.videoFramesContext.lastFrameIdsMutexes[sourceID].lock();
         const auto frameId = ++context.videoFramesContext.lastframeIds[sourceID];
         context.videoFramesContext.lastFrameIdsMutexes[sourceID].unlock();
+
+        // Stop reborning when frame ID reached to input queue size
+        if (!context.isVideo && frameId >= FLAGS_n_iqs)
```

Comment:
Why? It just decreases the actual queue size

Reply:
From my understanding, the number of frames passing through each channel will be specified by `FLGAS_n_iqs`.  This code just ensure stopping reborning new frame when frame ID reached to `FLAGS_n_iqs`.

Reply:
> Why? It just decreases the actual queue size

From my understanding, the number of frames passing through each channel will be specified by `FLGAS_n_iqs`.  This code just ensure stopping reborning new frame when frame ID reached to `FLAGS_n_iqs`.